src/config/documentTypes.ts

```
@@ -0,0 +1,56 @@
+export const documentTypes = [
+  { 
```

ESLint

[error] Replace `··{·` with `↹{`

(prettier/prettier)